setup/Sandbox/sandbox.rst

```diff
@@ -52,11 +60,23 @@ extension. The JupyterLab interface also supports plain text and Markdown files.
 
 To learn more about JupyterLab, visit the `JupyterLab Documentation`_.
 
-.. note::
-   Changes made to Jupyter notebooks in the DEA Sandbox may be automatically 
-   overwritten as part of the automatic update process. To avoid this, we
-   recommend advanced users use Git to clone a new copy of ``dea-notebook``
-   into the Sandbox (`see guide here`_).
+Some things to know about The Sandbox
+-------------------------------
+
+The Sandbox is not a production environment and should be used for protyping and exploring
+DEA's data and tools. Changes made to Jupyter notebooks in the DEA Sandbox may be automatically 
```

Comment:
This still seems more like a general "be cautious" disclaimer than a more serious warning that any changes to the default notebooks will be overwritten every time a user restarts their server. I think this is fine for now, but at some point we should try and find a good place for that more specific warning (perhaps somewhere on the Sandbox side where it's impossible to miss)

Reply:
(I'm still haunted by losing an entire day of work to this back in the early days of the Sandbox, and suspect it may be a more common issue for new users than we realise)